addons/circulation/functions/fnc_checkPulseLocal.sqf

```diff
@@ -0,0 +1,61 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Glowbal
+ * Local callback for checking the pulse or heart rate of a patient.
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ * 2: Body Part <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, cursorObject, "Head"] call ace_medical_treatment_fnc_checkPulseLocal
+ *
+ * Public: No
+ */
+
+params ["_medic", "_patient", "_bodyPart"];
+
+private _heartRate = 0;
+
+if !([_patient, _bodyPart] call ACEFUNC(medical_treatment,hasTourniquetAppliedTo)) then {
+    _heartRate = switch (true) do {
+        case (alive _patient): {
+            GET_HEART_RATE(_patient)
+        };
+        case (alive (_patient getVariable [QACEGVAR(medical,CPR_provider), objNull])): {
+            random [25, 30, 35] // fake heart rate because patient is dead and off state machine
```

Comment:
shouldnt that show bigger number? 100-110 or smth like that we have defined for CPR?

Reply:
we dont have a heartrate defined for CPR, plus you are not going to get an accurate pulse rate when CPR is happening

Reply:
of course we have https://github.com/KAT-Advanced-Medical/KAM/blob/20aef94aa9019de2657a6d78b011181108c17d5f/addons/vitals/functions/fnc_handleCardiacFunction.sqf#L36